### PR TITLE
Improved UI Of Config Activity

### LINKED ISSUE #1973: Improving the UI of generate config file activity
**Actual Behaviour**

UI of generate config file doesn't look ordered. Lot of empty space on screen

**Expected Behaviour**

Checkboxes should be assigned, more screen space should be covered

**Steps to reproduce it**

Open Generate config file from nav bar.

**LogCat for the issue**

N/A

**Screenshots of the issue**

Screenshot_20190909-121635_PSLab

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by kshitiz-kumar**
Fixes #1973

**Changes**: The Config Activity UI was unordered, so changes were made in the config layout by adding a number of views.

**Screenshot/s for the changes**: 
<image src = "https://user-images.githubusercontent.com/38812752/65950873-8fab4080-e45c-11e9-879e-fbd7b06c0d89.jpeg" width=250>

<image src ="https://user-images.githubusercontent.com/38812752/65950876-90dc6d80-e45c-11e9-8c99-0f199cd938b9.jpeg" width=250>

WhatsAppVideo2019-10-01at24050PM

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app-fdroid-debug.zip](https://github.com/fossasia/pslab-android/files/3684930/app-fdroid-debug.zip)

### CODE REVIEW

**Comment by kshitiz-kumar** [comment]
@CloudyPadmal please review.

File: app/src/main/java/io/pslab/activity/CreateConfigActivity.java [hunk 1]
```diff
@@ -134,17 +139,17 @@ private void createArrayLists() {
...
-        for (int i = 0; i < params.length; i++){
-            CheckBox checkBox = new CheckBox(CreateConfigActivity.this);
-            checkBox.setText(params[i]);
-            LinearLayout.LayoutParams checkBoxParams = new LinearLayout.LayoutParams(LinearLayout.LayoutParams.WRAP_CONTENT,LinearLayout.LayoutParams.WRAP_CONTENT);
-            checkBoxParams.gravity = Gravity.CENTER_HORIZONTAL;
-            checkBoxParams.setMargins(0,(int)getResources().getDimension(R.dimen.create_config_margin1),0,0);
-            checkBox.setLayoutParams(checkBoxParams);
-            paramsListContainer.addView(checkBox, i);
+        for (int i = 0; i < params.length; i++) {
+
```
Comment (ho-dor) [inline app/src/main/java/io/pslab/activity/CreateConfigActivity.java]: remove extra lines.

File: app/src/main/java/io/pslab/activity/CreateConfigActivity.java [hunk 2]
```diff
@@ -134,17 +139,17 @@ private void createArrayLists() {
...
+            CheckBoxGetter check = new CheckBoxGetter(params[i], false);
+            list.add(check);
         }
+        CheckBoxAdapter box;
+        box = new CheckBoxAdapter(CreateConfigActivity.this, list);
+        paramsListContainer.setAdapter(box);
+
```
Comment (ho-dor) [inline app/src/main/java/io/pslab/activity/CreateConfigActivity.java]: remove extra lines.

**Comment by CloudyPadmal** [comment]
Can we reduce the margin between each checkbox item in list view? Currently it looks a bit much..

**Comment by neel1998** [comment]
As mentioned in the #1960 it would be better to add scroll to the entire layout and not just the parameters section. @kshitiz-kumar

**Comment by kshitiz-kumar** [comment]
> Can we reduce the margin between each checkbox item in list view? Currently it looks a bit much..

Yeah sure. I can give it a try

**Comment by kshitiz-kumar** [comment]
> As mentioned in the #1960 it would be better to add scroll to the entire layout and not just the parameters section. @kshitiz-kumar
Scroll is added to entire layout also. But I think it can be further optimised.  I am working on it.

**Comment by kshitiz-kumar** [comment]
@CloudyPadmal @neel1998  the required changes are made. The margin between checkboxes removed and the apk been updated. I think things are working fine. 
>>Screen shot 

<image src="https://user-images.githubusercontent.com/38812752/66107456-f6a43300-e5dd-11e9-85f0-6794fd52954d.jpeg" width =300>

**Comment by kshitiz-kumar** [comment]
@CloudyPadmal please review.

**Comment by kshitiz-kumar** [comment]
@CloudyPadmal  please review.

**Pull request merged**